data/data-pipeline/data_pipeline/etl/sources/cdc_svi_index/etl.py

```diff
@@ -0,0 +1,138 @@
+
+        # New file name
+        tmp_svi_csv_file_path = (
+            self.CDC_SVI_INDEX_ZIP_FILE_DIR / "SVI2018_US.csv"
+        )
+
+        self.df = pd.read_csv(
+            filepath_or_buffer=tmp_svi_csv_file_path,
+            dtype={self.CDC_SVI_INDEX_TRACTS_FIPS_CODE: object},
+        )
+
+        # Note: In this dataset all US census tracts are ranked against one another.
+        # Puerto Rico is not included in this dataset
+        self.df.rename(
+            columns={
+                self.CDC_SVI_INDEX_TRACTS_FIPS_CODE: self.GEOID_TRACT_FIELD_NAME,
+                "RPL_THEME1": field_names.CDC_SVI_INDEX_SE_THEME_PERCENTILE_FIELD,
+                "RPL_THEME2": field_names.CDC_SVI_INDEX_HOUSEHOLD_THEME_PERCENTILE_COMPOSITION_FIELD,
+                "RPL_THEME3": field_names.CDC_SVI_INDEX_LANGUAGE_THEME_PERCENTILE_FIELD,
+                "RPL_THEME4": field_names.CDC_SVI_INDEX_HOUSING_TRANSPORTATION_PERCENTILE_FIELD,
+                "RPL_THEMES": field_names.CDC_RPL_THEMES_OVERALL_PERCENTILE_FIELD,
+                "SPL_THEME1": field_names.CDC_SVI_INDEX_SE_THEME_SOS_FIELD,
+                "SPL_THEME2": field_names.CDC_SVI_INDEX_HOUSEHOLD_THEME_SOS_FIELD,
+                "SPL_THEME3": field_names.CDC_SVI_INDEX_LANGUAGE_THEME_SOS_FIELD,
+                "SPL_THEME4": field_names.CDC_SVI_INDEX_HOUSING_TRANSPORTATION_SOS_FIELD,
+                "SPL_THEMES": field_names.CDC_SVI_INDEX_THEMES_OVERALL_SOS_FIELD,
+                "F_TOTAL": field_names.CDC_SVI_INDEX_THEMES_OVERALL_TOTAL_FIELD,
+            },
+            inplace=True,
+            errors="raise",
+        )
+
+        #  At or above 90 for percentile rank
+        self.df[field_names.CDC_SVI_INDEX_THEMES_PRIORITY_COMMUNITY] = (
+            self.df[field_names.CDC_RPL_THEMES_OVERALL_PERCENTILE_FIELD]
+            >= self.CDC_RPL_THEMES_THRSHOLD
+        )
+
+        expected_census_tract_field_length = 11
+        if (
+            not self.df[self.GEOID_TRACT_FIELD_NAME]
+            .apply(lambda x: len(str(x)))
+            .eq(expected_census_tract_field_length)
+            .all()
+        ):
+            raise ValueError(
+                f"GEOID Tract must be length of {expected_census_tract_field_length}"
+            )
+
+    def load(self) -> None:
+        logger.info("Saving CDC SVI Index Data")
+
+        # The F-prefixed variables symbolize a flag (as defined in the documentation)
+        # Tracts in the top 10%, i.e., at the 90th percentile of values, are given a value of 1 to indicate high vulnerability.
+        # Tracts below the 90th percentile are given a value of 0.
+        # For a given theme, the flag value is the number of flags for variables comprising the theme. The overall
+        # flag value is calculated for each tract as the number of all variable flags (F_TOTAL)
+        list_of_flag_columns = [
+            x for x in self.df.columns if str(x).startswith("F_")
+        ]
+
```

Comment:
These columns were an afterthought and I did not know where else to place them. There was nowhere else where I felt I could define them, so I thought this would be best. I welcome feedback.